packages/components/alert-dialog/src/AlertDialogContent.tsx

```diff
@@ -37,7 +38,9 @@ export const AlertDialogContent = forwardRef<AlertDialogContentElement, AlertDia
           ref={ref}
           data-spark-component="alert-dialog-content"
           {...others}
+          className={cx(className, '!w-auto min-w-sz-288')}
```

Comment:
In the `Dialog` component the width is set to full. I can think of two options here:
- Override it
- Create a new prop for the `Dialog` and make the full width be optional (default to full width)

WDYT ?

Reply:
2nd

Reply:
👍🏻